src/test/java/com/wikia/webdriver/testcases/desktop/mcfootertests/NonEnAnonMixedContentFooterTests.java

```diff
@@ -107,6 +207,19 @@ void isUserTakenToDiscussionsAfterClickOnViewAll() {
     Assertion.assertTrue(discussions.isDiscussions());
   }
 
+  @RunOnly(language = "szl")
+  @Test
+  void isUserTakenToDiscussionsAfterClickOnViewAllSzl() {
+    DiscussionsPage discussions = new MixedContentFooter().openWikiMainPage()
+        .scrollToMCFooter()
+        .getDiscussionsCard()
+        .scrollToDiscussions()
+        .clickOnViewAllLinkInDiscussions();
+
+    Assertion.assertTrue(discussions.isDiscussions());
```

Comment:
Would be awesome to renamed it to something more descriptive like `isDiscussionsPresent()` or similar